src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.liquid

```diff
@@ -252,7 +252,7 @@
 {%                     elseif parameter.IsArray -%}
                     foreach (var item_ in {{ parameter.VariableName }})
                     {
-                        content_.Add(new System.Net.Http.StringContent(item_), "{{ parameter.Name }}");
+                        content_.Add(new System.Net.Http.StringContent(ConvertToString({{ parameter.VariableName }}, System.Globalization.CultureInfo.InvariantCulture)), "{{ parameter.Name }}");
```

Comment:
Shouldnt this be `ConvertToString(item_,...`?

Reply:
Well, thanks for the fast response -> fixed